src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -1058,6 +1059,15 @@ public ChangelogCommand includes(ObjectId rev) {
                 return includes(rev.name());
             }
 
+            public ChangelogCommand firstParent() {
```

Comment:
Please add `@Override`.

Reply:
thanks for catching this, I addressed this in my latest commit